src/events/math.js

```diff
@@ -0,0 +1,103 @@
+import operate from './calculate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] As mentioned by the previous code reviewer, This operation still needs some fix 😄. I got a blank screen when I clicked on 2 and then clicked *, then clicked the `AC` instead of getting 0, Please let ensure that whenever the `AC` button is clicked it returns `0`. You can create a conditional statement in your `handleClick` method so that when the button clicked is `AC` it returns `0`.

Reply:
Hello, thank you for your review, I have fixed the operation and now I can get 0,